editor/editor_plugin.cpp

```diff
@@ -35,14 +35,14 @@
 #include "editor/editor_paths.h"
 #include "editor/editor_resource_preview.h"
 #include "editor/editor_settings.h"
-#include "editor/editor_undo_redo_manager.h"
 #include "editor/export/editor_export.h"
 #include "editor/filesystem_dock.h"
 #include "editor/plugins/canvas_item_editor_plugin.h"
 #include "editor/plugins/node_3d_editor_plugin.h"
 #include "editor/plugins/script_editor_plugin.h"
 #include "editor/project_settings_editor.h"
 #include "editor/scene_tree_dock.h"
+#include "editor_undo_redo_manager.h"
```

Comment:
Everywhere else it included as `editor/editor_undo_redo_manager.h`. Accidental change?

Reply:
Yes, fixed.